modules/quanthub_chat_overlay/quanthub_chat_overlay.libraries.yml

```diff
@@ -0,0 +1,5 @@
+chat-overlay:
+  js:
+    /libraries/quanthub-drupal-react/quanthub-libs/dist/chat-overlay/index.mjs: { attributes: { type: module } }
```

Comment:
The path exists in local repo, and not in this one. It can't be used here without adding it to this repo.